src/sentry/static/sentry/app/components/dateTime.tsx

```diff
@@ -5,7 +5,16 @@ import _ from 'lodash';
 
 import ConfigStore from 'app/stores/configStore';
 
-class DateTime extends React.Component {
+type Props = {
+  date: any;
```

Comment:
Even though the propTypes is any, we should be a bit more specific on the type. It looks like it can be either: 1) string, 2) number (e.g. unix timestamp), or 3) Date object

Reply:
Let's use `moment.MomentInput`. I suspect the coercedDate line might be unnecessary and can be deleted as moment can handle string and unix time as input.